infrastructure: moneropulse spelling and grammar (#25)
monero-project/monero-docs: docs/en/infrastructure/monero-pulse.md

## Fixing "WARNING: no two valid MoneroPulse DNS checkpoint records were received"

> This means that the DNS server that you are using does not acknowledge the +dnssec flag necessary to securely query for checkpoints.

Reviewer: ```suggestion
This means that the DNS server you're using doesn't acknowledge the `+dnssec` flag which is necessary to securely query for checkpoints.
```
